src/ert/gui/simulation/view/realization.py

```diff
@@ -80,7 +80,9 @@ def paint(self, painter, option: QStyleOptionViewItem, index: QModelIndex) -> No
         painter.drawRect(option.rect)
 
         painter.setPen(QPen(QColorConstants.Black))
-        percentage_done = int((finished_count * 100.0) / total_count)
+        percentage_done = int(
+            (finished_count * 100.0) / (total_count if total_count > 0 else 1)
```

Comment:
Shouldn't 0 out of 0 completed be 100% ?

Reply:
Sure. That makes also 0 out of 0 sense.

Reply:
https://en.wikipedia.org/wiki/Vacuous_truth

Reply:
```
 percentage_done = (
            100 if total_count < 1 else int((finished_count * 100.0) / total_count)
        )
```